__tests__/pattern/pattern.query.test.js

```diff
@@ -0,0 +1,349 @@
+const { gql } = require('apollo-boost');
+/* eslint-disable no-undef */
+const client = require('../../utils/apollo-test-client');
+require('dotenv').config();
+const {
+  patternDoesNotExistId,
+  skip,
+  limit,
+  wrongLimit,
+  wrongSkip,
+  languageTypeName,
+  imageTypeName,
+} = require('./pattern.variables');
+const { PATTERN_NOT_FOUND } = require('../../error-messages/pattern.messages');
+
+let patternId;
+
+const testValue = 't e s t new';
+const patternToAdd = {
```

Comment:
Why does this code duplicate the code inside the `pattern.variables.js`?

Reply:
thats different testValue for query and mutations